doc/FAQ.org

```diff
@@ -268,8 +268,13 @@ You can modify the syntax table of the mode in question. For example, for Python
 mode:
 
 #+BEGIN_SRC emacs-lisp
-(with-eval-after-load 'python
-  (modify-syntax-entry ?_ "w" python-mode-syntax-table))
+;; In your dotspacemacs/user-init()
```

Comment:
I think this comment should be placed out of the code block, in the org text. You can modify the sentence just above to remove the explicit python mention, and add there the information where it should be placed. Also this would probably better be placed in `dotspacemacs/user-config` (I haven't tested it though)